app/src/main/res/layout-land/fragment_top_up_success.xml

```diff
@@ -4,16 +4,14 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/styleguide_blue"
-    android:orientation="horizontal"
-    >
+    android:background="@color/styleguide_blue">
 
   <com.airbnb.lottie.LottieAnimationView
       android:id="@+id/top_up_success_animation"
       android:layout_width="0dp"
       android:layout_height="0dp"
-      android:layout_marginBottom="64dp"
       android:visibility="visible"
+      app:layout_constraintDimensionRatio="H,1000:592"
```

Comment:
Was this ratio added intentionally? 
On my device, the success animation seems to not be centered (only on landscape)

Reply:
its to keep the ratio. its the dimensions on the json of the animation